### PR TITLE
[ADD] Coinsamatik: Add addons-vauxoo and stock_logistic_workflow T#54677

### PR DESCRIPTION
**Pull request by geraspace**
- Add addons-vauxoo and stock_logistic_workflow submodules

### CODE REVIEW

**Comment by geraspace** [comment]
Hello, Bruce.

Here's the update with the added submodules. Let me know if something's needed. 

@BruceSOT

File: .gitmodules [hunk 1]
```diff
@@ -30,3 +30,9 @@
 	path = vendor-bills
 	url = git@git.vauxoo.com:vauxoo/vendor-bills.git
 	branch = 13.0
+[submodule "addons-vauxoo"]
+	path = addons-vauxoo
+	url = git@github.com:Vauxoo/addons-vauxoo.git
```
Comment (luiseevaquer) [inline .gitmodules:35]: @geraspace 

Add `branch = 13.0`

File: .gitmodules [hunk 2]
```diff
@@ -30,3 +30,9 @@
...
+[submodule "stock-logistics-workflow"]
+	path = stock-logistics-workflow
+	url = git@github.com:OCA/stock-logistics-workflow.git
```
Comment (luiseevaquer) [inline .gitmodules:39]: @geraspace 

Add it here too

**Comment by geraspace** [comment]
@lescobarvx thanks for the review.

Should be correct now.

**Pull request closed without merging**